controllers/v1/submissionsController.js

```diff
@@ -2233,4 +2245,158 @@ module.exports = class Submission extends Abstract {
     })
   }
 
+    /**
+  * @api {get} /assessment/api/v1/submissions/delete/:submissionId Delete Submission.
+  * @apiVersion 1.0.0
+  * @apiName Delete Submission
+  * @apiGroup Submissions
+  * @apiSampleRequest /assessment/api/v1/submissions/delete/5c6a352f77c8d249d68ec6d0
+  * @apiParamExample {json} Response:
+  * {
+  *    "message": "Submission deleted successfully",
+  *    "status": 200
+  *  }
+  * @apiUse successBody
+  * @apiUse errorBody
+  */
+
+   /**
+   * Delete submission.
+   * @method
+   * @name delete
+   * @param {String} req.params._id - submission id.
+   * @returns {JSON} - status and deleted message.
+   */
+
+  async delete(req) {
+    return new Promise(async (resolve, reject) => {
+
+      try {
+            
+          let submissionData = await submissionsHelper.delete(
```

Comment:
@aman-tunerlabs - User can delete only his submissions and which are not started.